pornography-hosts

```diff
@@ -15636,5 +15636,21 @@
 0.0.0.0 zzbabes.com
 0.0.0.0 zzpornpics.com
 0.0.0.0 zzztube.com
+
+0.0.0.0 ixiporn.com
+0.0.0.0 lustmaza.com
+0.0.0.0 dropmaza.com
+0.0.0.0 artsporn.com
+0.0.0.0 sexfullmovies.com
+0.0.0.0 exmovies.co.in
+0.0.0.0 teensexmix.com
+0.0.0.0 indiansexmms.info
+0.0.0.0 rajwap.me
+0.0.0.0 smutindia.pro
+0.0.0.0 pinkheartmovies.xyz
+0.0.0.0 ggmaal.cyou
+0.0.0.0 kashtanka.tv
+0.0.0.0 lustesthd.com
```

Comment:
https://mypdns.org/my-privacy-dns/porn-records/-/issues/1452

Reply:
> Hi! May I know what's going on

See https://github.com/Sinfonietta/hostfiles/pull/156#discussion_r769902529

You might start to understand why I'm attending to the COPYRIGHT and LICENSE as there 2 very specific users who clearly have HUGE personal issues and trying by all means to avoid giving proper credit and only steal my work, here @Sinfonietta project is one of the tools they are using, sadly as I have no beefs with @Sinfonietta.

Lately is have been looking like a tendency this repo have pretty much like it is used to redirects my work into SB hosts through this project. That's why, when I find these high identical matches I attend to it. You @defa02005 and @cityofships (Exact match) just happens to be that 1 to many with near if not exactly identical domains as I have done just almost previously.

PS: Very happy to see your replies and actual questions vs other current open PR (https://github.com/Sinfonietta/hostfiles/pull/159) who just avoid responding and blacklisting me.

Don't be wrong, I'm a huge fan of sharing, but it comes with a thank you to `@user`...

> Coz I think your list also merges in the StevenBlack one

As mentioned, they do what they can to avoid giving proper credit. but currently he is using a  very outdated copy and if I remember it right , then he is missing about 1000 records from the current published list

![image](https://user-images.githubusercontent.com/44526987/146247916-9dace196-1ec8-4319-8ad5-2e5afe842993.png)

https://github.com/StevenBlack/hosts/pull/1784

----------------

As the lit all ready is of, how do you use this project?


-------------------

If you hit me up on either keybase or mypdns and can give you the reason sb don't like, for dmntx nobody knows, and I honestly doubt he can give a reason him self.